vscode-lean4/src/abbreviation/abbreviations.json

```diff
@@ -761,7 +781,7 @@
     "sphericalangle": "∢",
     "section": "§",
     "searrow": "↘",
-    "setminus": "∖",
+    "setminus": "\\",
```

Comment:
This one feels slightly weird to me; we may not (currently) use this character in mathlib, but this really is "setminus"!

Reply:
I think it's most important that `\setminus` expands to Mathlib's character for set differences, even if that character is "wrong". I don't actually care which symbol is used for set differences. If you're willing to change the character used in Mathlib, I'm happy to revert this (in a follow-up PR).